lib/mappings/bib-mapping.json

```diff
@@ -1155,7 +1147,8 @@
       },
       {
         "marc": "653",
-        "description": "Index Term"
+        "description": "Index Term",
```

Comment:
This is unrelated to the rest of this work, but I saw that we do have a switch for nypl source. This field is a locally defined subject, and other institutions seem to use it for purposes other than display